crates/binjs_io/src/bytes/compress.rs

```diff
@@ -32,12 +35,16 @@ pub enum Compression {
     Lzw,
 }
 
-impl Rand for Compression {
-    fn rand<R: Rng>(rng: &mut R) -> Self {
+impl Distribution<Compression> for Standard {
+    fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Compression {
         use self::Compression::*;
-        rng.choose(&[Identity, Gzip, /* Deflate is apprently broken https://github.com/alexcrichton/flate2-rs/issues/151 , */ Brotli /*, Lzw doesn't work yet */])
-            .unwrap() // The array is not empty.
+        let choices = [Identity, Gzip, /* Deflate is apprently broken https://github.com/alexcrichton/flate2-rs/issues/151 , */ Brotli /*, Lzw doesn't work yet */];
+        choices.choose(rng)
+            .unwrap()
             .clone()
+        // rng.choose(&[Identity, Gzip, /* Deflate is apprently broken https://github.com/alexcrichton/flate2-rs/issues/151 , */ Brotli /*, Lzw doesn't work yet */])        
+        //     .unwrap()
```

Comment:
Could you remove that code instead of commenting it out?

Reply:
My bad 👍